OpenRA.Mods.RA/Traits/LeavesHusk.cs

```diff
@@ -48,11 +76,19 @@ public void Killed(Actor self, AttackInfo e)
 					new ParentActorInit(self),
 					new LocationInit(self.Location),
 					new CenterPositionInit(self.CenterPosition),
-					new OwnerInit(self.Owner),
-					new RaceInit(race),
-					new SkipMakeAnimsInit()
+					new RaceInit(race)
```

Comment:
I don't think you can cache `race` safely any more. Not with the husk's owner being modifiable.